docs/management_commands.rst

```diff
@@ -0,0 +1,74 @@
+Django Management Commands
+==========================
+
+Export Command
+--------------
+
+The ``export`` command allows you to export data from a specified Django model
+or a resource class. The exported data can be saved in different formats, such
+as CSV or XLSX.
+
+Usage
+-----
+
+.. code-block:: bash
+
+    python manage.py export <format> <resource> [--encoding ENCODING]
+
+- **format**: Specify the format in which the data should be exported. -
+- **resource**: Specify the resource or model to export. Accepts a resource class or a model class in dotted path format. - **--encoding** (optional): Specify the encoding (e.g., 'utf-8') to be used for the exported data.
+
+Example
+-------
+
+.. code-block:: bash
+
+    python manage.py export CSV auth.User
+
+This command will export the User model data in CSV format using utf-8
+encoding.
+
+Another example:
+
+.. code-block:: bash
+
+    python manage.py export XLSX mymodule.resources.MyResource
+
+This command will export the data from ``MyResource`` resource in XLSX format
+using utf-8 encoding.
+
+Import Command
+--------------
+
+The ``import`` command allows you to import data from a file into a specified
+Django model or a custom resource class.
+
+Usage
+-----
+
+.. code-block:: bash
+
+    python manage.py import <resource> <import_file_name> [--format FORMAT] [--encoding ENCODING] [--dry-run] [--raise-errors]
+
+- **resource**: The resource class or model class in dotted path format.
+- **import_file_name**: The file from which data is imported (``-`` can be used to indicate stdin).
+- **--format** (optional): Specify the format of the data to import. If not provided, it will be guessed from the mimetype.
```

Comment:
I noticed that this is case sensitive - ie. has to be 'CSV' not 'csv'.

Reply:
Yes, this is defined here:

https://github.com/bmihelac/django-import-export/blob/feat/management-commands/import_export/command_utils.py#L53

```
fallback_format_name = f"import_export.formats.base_formats.{format_name}"
```

I could uppercase `format_name`, but it felt wrong."

Reply:
ok - I wonder if we should make it clear in the help comment for `format` that it must match the case of the defined format.  I think users without knowledge of the internals will think they can supply 'csv' lower case as the argument.

Reply:
Make sense - added another fallback that uppercase format name if not found in 2a17796